README.rst

```diff
@@ -3,6 +3,9 @@ codespell
 
 Fix common misspellings in text files. It's designed primarily for checking
 misspelled words in source code, but it can be used with other files as well.
+It does not check for word membership in a complete dictionary, but instead
+looks for a set of common misspellings. Therefore it might catch errors like
+"adn", but it will not catch "adnasdfasdf".
 
```

Comment:
Derp. Talk about circular dependency. Using Codespell to check readme of repo finds spelling error, which only exists to show example of spelling error. Not sure if I should fix this.

Reply:
Well at least we know it's working @tir38 ! Joking apart we could either ignore the line or add the specific typo to an allowed list (although if we do the latter it would be better if it was a more obscure word/less likely typo, but perhaps that defeats the point of the example; `abandonned` is our goto typo, but it's not as obvious as the example one).

@larsoner Travis didn't catch this as it's only checking the module source stuff, not the surrounding files:
`codespell --skip="codespell_lib/tests/test_basic.py,codespell_lib/data/*" codespell_lib/`

Reply:
Feel free to add it to the checks

Reply:
Done in #1596

Reply:
See https://github.com/codespell-project/codespell/pull/1535#issuecomment-782903268 for more detail of fixes.